### DISCUSSION TITLE
Lagspike in the first 30 seconds of playing.

### DISCUSSION THREAD

**Discussion by JuLi0n21** [ppy/osu#22882] [Q&A]
### Type

Performance

### Bug description

I get a lag spike in the first 30 seconds every time i play a map. it doesnt matter if i play mulithread / singlecore or limit the fps.

I sometimes get a Lag at anypart.

This has been an issue for multiple patches. 

I also have 22k beatmaps 

### Screenshots or videos

 [Lag at 0:28](https://youtu.be/5ohpJKgTBH4)

[Lag at 0:32](https://youtu.be/oafHn3cu3AE)

[Lag at 0:31 and 0:47](https://youtu.be/vPme_o8IFWI)

### Version

2023.305.0-lazer

### Logs

[performance.log](https://github.com/ppy/osu/files/10991175/performance.log)
[database.log](https://github.com/ppy/osu/files/10991176/database.log)
[input.log](https://github.com/ppy/osu/files/10991177/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/10991178/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/10991180/network.log)
[performance-audio.log](https://github.com/ppy/osu/files/10991182/performance-audio.log)
[updater.log](https://github.com/ppy/osu/files/10991188/updater.log)
[runtime.log](https://github.com/ppy/osu/files/10991187/runtime.log)
[performance-update.log](https://github.com/ppy/osu/files/10991185/performance-update.log)
[performance-draw.log](https://github.com/ppy/osu/files/10991183/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/10991184/performance-input.log)

**Comment by peppy** [comment]
The frame graphs make it look like something else on your PC causing the issue.

https://osu.ppy.sh/help/wiki/Performance_Troubleshooting has some tips you can follow.

**Comment by JuLi0n21** [comment]
The Video qualitiy is god awful compared to the actual ingame experience, everything runs without issues except a spike running from a few milliseconds to almost a second, the inputs still seem to go thorugh but having the screen frezze is very annoing also runing a local build didnt change anything.
